### ISSUE TITLE
MultiSelect: Form elements must have labels accessibility violation

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#513]
Install the Axe Dev Tools for Accessibility plug-in.
Navigate to https://v17.primeng.org/multiselect on Chrome browser
Scan the page with Axe

Observe 'Form elements should have labels' error reported for hidden input

![Image](https://github.com/user-attachments/assets/7bed3200-1e14-4fa4-83df-801e23dedab7)

Angular version: 18.2.9
PrimeNG version: v17
Node version: 18.19.1

> Migrated from `primefaces/primeng#18183` — originally by `@elena-ung` on 2025-04-28

**Comment by geromegrignon** [comment]
*[Originally posted by `@breakone` on 2025-07-21T07:28:20.000Z]*

`@elena-ung` I think, you can add an `ariaLabel` to prevent this issue:

```
<p-multiselect
  [ariaLabel]="'my label'"
  ...
```

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-01-18T00:24:26.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
